### ISSUE TITLE
[Feature Request] Inventory player model and examine rotation

### ISSUE THREAD

**Issue by juuuulian-s** [nipkownix/re4_tweaks#575]
### Describe the feature you'd like to see added

It'd be cool to see an option to allow the player to rotate Leon around while in the inventory screen, as well as a free rotation feature for the examine item view, similar to the bottlecap rotation, or the examine feature in other RE games. We're not using the right stick in those screens anyways... No practical reason, it'd just be fun to get an alternate view of all the different weapon equip stances and item models (particularly with the HD Project sprucing up all those models and textures)

Also since I'd rather not clutter up the Issues tab I'll bundle another, completely unrelated feature request here:
Make the Punisher ignore metal helmets/armor. Maybe as part of the exclusive upgrade, but TBH the base gun just does not measure up to the other pistols, and while it can be used effectively, its gimmick is not as versatile or advantageous as it could be.

**Comment by emoose** [comment]
Nice idea, looks like pointer to the inventory character `cModel` is kept inside `g_pPl` pointer (value of 0xC7BA6C in 1.1.0), editing the `ang_A0.x/.y/.z` fields of that seems to rotate them on inventory fine (float is at `model_addr + 0xA0/0xA4/0xA8`)

Would probably be possible to change the `pos_94` value to move the chara around too, if the inventory box model could be made invisible maybe could work as a kind of character-examine mode.

The item examine code seems to have code to let you control it already too, but it goes unused when examining normal items, maybe gets used for examining bottle-caps since that code does mention a `cap_dist_max` value.

Forcing that code to run does let you move normal items around, but the default spin anim is still applied at same time, not sure how to disable that yet.

Guess how this could work is picking up item would display normal spinning anim, but moving one of the sticks would disable that anim and let you rotate the item around, then if you release stick for N seconds it would resume spinning.

**Comment by juuuulian-s** [comment]
Yeah kinda like how the main menu background rotation works currently. That would be pretty neat
